### PR TITLE
Add Advanced Features from Buzzard Classic

### PR DESCRIPTION
**Pull request by NPoole**
- Adds "Advanced" Checkbox which reveals advanced options pane
- Adds Inline Formatting from Buzzard Classic
- Adds Overlining using KiCad syntax and overline style selection
- Adds UI buttons for commonly used symbols

### CODE REVIEW

File: KiBuzzard/dialog/dialog.py [hunk 1]
```diff
@@ -58,7 +62,12 @@ def __init__(self, parent, config, buzzard, func):
         self.m_LayerComboBox.SetSelection(0)
         self.m_HeightUnits.SetLabel("mm")
         self.m_WidthUnits.SetLabel("mm")
+<<<<<<< Updated upstream
```
Comment (arturo182) [inline KiBuzzard/dialog/dialog.py:65]: Merge leftovers

File: KiBuzzard/dialog/dialog.py [hunk 2]
```diff
@@ -74,6 +83,18 @@ def __init__(self, parent, config, buzzard, func):
 
         self.loadConfig()
 
+<<<<<<< Updated upstream
```
Comment (arturo182) [inline KiBuzzard/dialog/dialog.py:86]: Here

File: KiBuzzard/dialog/dialog.py [hunk 3]
```diff
@@ -223,6 +244,7 @@ def ReGeneratePreview(self, e=None):
 
         self.buzzard.alignment = self.m_AlignmentChoice.GetStringSelection()
 
+<<<<<<< Updated upstream
```
Comment (arturo182) [inline KiBuzzard/dialog/dialog.py:247]: And here

File: KiBuzzard/dialog/dialog.py [hunk 4]
```diff
@@ -266,8 +296,20 @@ def ReGeneratePreview(self, e=None):
             if requestedWidth<scaledLabelWidth:
                 self.m_WidthCtrl.SetValue(scaledLabelWidth)
 
+<<<<<<< Updated upstream
```
Comment (arturo182) [inline KiBuzzard/dialog/dialog.py:299]: And here

File: KiBuzzard/dialog/dialog_text_base.py [hunk 5]
```diff
@@ -297,12 +399,29 @@ def __init__( self, parent ):
         # Connect Events
         self.Bind( wx.EVT_INIT_DIALOG, self.OnInitDlg )
         self.m_MultiLineText.Bind( wx.EVT_CHAR_HOOK, self.OnCharHook )
+<<<<<<< Updated upstream
```
Comment (arturo182) [inline KiBuzzard/dialog/dialog_text_base.py:402]: Also here

File: KiBuzzard/dialog/dialog_text_base.py [hunk 6]
```diff
@@ -313,4 +432,32 @@ def OnCharHook( self, event ):
     def OnOkClick( self, event ):
         pass
 
+    def lineoverStyleChange( self, event ):
+        pass
+
+<<<<<<< Updated upstream
```
Comment (arturo182) [inline KiBuzzard/dialog/dialog_text_base.py:438]: Lastly, here

**Review by arturo182 — CHANGES_REQUESTED** [review]
Could you provide a screenshot of the UI changes?

File: KiBuzzard/dialog/dialog.py (hunk 1 above)
Comment (NPoole) [reply KiBuzzard/dialog/dialog.py:65]: Ah, I think this is GitHub Desktop's fault. I'll go fix things...

**Comment by NPoole** [comment]
Yes, here's a screenshot of the new UI. I'm also working on a PR right behind this one to add tool-tips.

With the "Advanced" box unchecked, the UI remains unchanged:
image

Checking the box reveals new options as follows: 
image

**Comment by NPoole** [comment]
Closing this PR to detangle the branch

**Pull request closed without merging**